Clarify default structures (#392)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-auth-scheme.md

The structure fields are defined as follows:

- "token_type" is a 2-octet integer, in network byte order, as described
above.

- "issuer_name" is a string containing the name of the issuer. This is a
hostname that is used to identify the issuer that is allowed to issue
tokens that can be redeemed by this origin. The string is prefixed with a
2-octet integer indicating the length, in network byte order.

> - "redemption_context" is an optionally empty field. If non-empty, it allows
> the origin to require that clients fetch tokens bound to a specific context, as
> opposed to reusing tokens that were fetched for other contexts. See
> {{context-construction}} for example contexts that might be useful in practice.
> When present, this value is a 32-byte context generated by the origin. Valid
> lengths for this field are either 0 or 32 bytes. The field is prefixed with a
> single octet indicating the length. Challenges with redemption_context values
> of invalid lengths MUST be ignored.

Author: ```suggestion
- "redemption_context" is a field that is either 0 or 32 bytes, prefixed with a single
octet indicating the length (either 0 or 32). If value is non-empty, it is a 32-byte value
generated by the origin that allows the origin to require that clients fetch tokens
bound to a specific context, as opposed to reusing tokens that were fetched for other
contexts. See {{context-construction}} for example contexts that might be useful in
practice. Challenges with redemption_context values of invalid lengths MUST be ignored.
```

> - "origin_info" is an optionally empty string containing one or more origin names,
> which allows a token to be scoped to a specific set of origins. The string
> is prefixed with a 2-octet integer indicating the length, in network byte
> order. If empty, any non-origin-specific token can be redeemed. If the string

Author: ```suggestion
- "origin_info" is a string that is either empty, or contains one or more origin
names that allow a token to be scoped to a specific set of origins. The string
is prefixed with a 2-octet integer indicating the length, in network byte
order. If empty, any non-origin-specific token can be redeemed. If the string
```
